### PR TITLE
Newsletter 2022 07

### PR DESCRIPTION
**Pull request by vdauwera**
Last call for edits? 

@nlharris I think this is ready to go? Sorry for the radio silence in August, I was out for most of it.

### CODE REVIEW

File: newsletters/2022-07.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+A report about BOSC 2022 is in progress, and will appear as part of the [BOSC collection on F1000](https://f1000research.com/collections/bosc). All BOSC presenters are encouraged to submit their presentation slides to the collection by September 30, 2022.
```
Comment (nlharris) [inline newsletters/2022-07.md:21]: their presentation slides and/or posters

File: newsletters/2022-07.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+**BOSC 2023** will take place in July as a COSI of [ISMB/ECCB 2023](https://www.iscb.org/ismbeccb2023) as a hybrid event; in-person in Lyon (France), and online.
+
+
+### OBF Event Fellowships (formerly "Travel Fellowships")
+
+Last round: April 1, 2022  
+Next round: October 1, 2022
+
+Our [call for applications is open again, with a **deadline on October 1st**](https://www.open-bio.org/event-awards/). We invite applications for financial support to attend or host events taking place between November 1, 2022 and November 1, 2023.
+
```
Comment (nlharris) [inline newsletters/2022-07.md:36]: Add link to blog post about call for applications for Oct 1 round? https://www.open-bio.org/2022/08/12/obf-event-fellowship-2022-round2/
(I'm not sure where best to add it)

**Review by nlharris — APPROVED** [review]
Looks good; I made two minor comments.

**Review by hlapp — APPROVED** [review]
See inline comments, but 👍

**Comment by nlharris** [comment]
I agree with Hilmar's suggest to make the link to the BOSC 2022 page rather than the BOSC page that changes every year.

As far as I'm aware, ISCB has not yet made the keynote videos available.

**Comment by nlharris** [comment]
Regarding the keynote videos, I just pinged ISCB and asked if they were ready yet, and Seth said they were getting close. I don’t think we should block on that (especially since the newsletter is dated July/August, it would be good to get it out before October!) but if that comes through before the newsletter is sent, I’ll let you know.

File: newsletters/2022-07.md (hunk 2 above)
Comment (vdauwera) [reply newsletters/2022-07.md:36]: done, worked it in

File: newsletters/2022-07.md (hunk 1 above)
Comment (vdauwera) [reply newsletters/2022-07.md]: done

File: newsletters/2022-07.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+In longer form: 
+
+## OBF News
+
+### BOSC News
+
+**[BOSC 2022](http://open-bio.org/events/bosc/)** took place in July as a COSI (Community of Special Interest) of [ISMB 2022](https://www.iscb.org/ismb2022), which was held as a hybrid event, in-person in Madison, WI (USA) and online. Over the two days of the BOSC COSI, approximately 200 people participated in [BOSC sessions](https://www.open-bio.org/events/bosc-2022/bosc-2022-schedule/), which included 28 talks chosen from submitted abstracts, 46 posters, and a [panel discussion, “Building and Sustaining Inclusive Open Science Communities”](https://www.open-bio.org/events/bosc-2022/bosc-2022-panel/). BOSC 2022 included [joint keynotes](https://www.open-bio.org/events/bosc-2022/bosc-2022-keynotes/) with two other COSIs. Jason Williams gave a BOSC / Education COSI keynote entitled "Riding the bicycle: Including all scientists on a path to excellence". A joint session with Bio-Ontologies featured a keynote by Melissa Haendel, “The open data highway: turbo-boosting translational traffic with ontologies.”
```
[the remark this answers is not in the document]
Comment (vdauwera) [reply newsletters/2022-07.md]: Great point about the URL, done.

I made sure to spell out COSI in the first line so I don't think we need to spell it out again for the BOSC instance.

**Comment by nlharris** [comment]
The BOSC report is now published on F1000Research: https://f1000research.com/articles/11-1034

File: newsletters/2022-07.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+A report about BOSC 2022 is in progress, and will appear as part of the [BOSC collection on F1000](https://f1000research.com/collections/bosc). All BOSC presenters are encouraged to submit their presentation slides and/or posters to the collection by September 30, 2022.
```
Comment (nlharris) [inline newsletters/2022-07.md:21]:
```suggestion
A [report about BOSC 2022](https://f1000research.com/articles/11-1034) has been published as part of the [BOSC collection on F1000](https://f1000research.com/collections/bosc). All BOSC presenters are encouraged to submit their presentation slides and/or posters to the collection by September 30, 2022.
```

Comment (nlharris) [reply newsletters/2022-07.md:21]: (I just learned about "suggesting mode" in PR reviews -- super-handy!

**Comment by vdauwera** [comment]
Ooh I like suggestion mode. 

Ok, I think this is good to go now, for real. Assuming it is, what is the next step? We merge the PR, then someone actually sends the newsletter? (Who and how exactly?)

**Comment by nlharris** [comment]
Yeah, let's give @peterjc a chance to review and merge, and Peter can also decide who should send it. (In the past, it was sent out by the newsletter editor, who was also an OBF Board member. This is the first time we've had a community volunteer put together the newsletter!)

**Comment by nlharris** [comment]
We have a Board hangout this Friday so if Peter hasn't looked at it by then I can bring it up.

File: newsletters/2022-07.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
+---
+tags: obf
+---
+
+_Having trouble viewing this email or looking for a permalink? [View this newsletter in your browser](https://github.com/OBF/newsletter/blob/master/newsletters/2022-07.md)._
+
+_This email was sent to the [bosc-announce](https://groups.google.com/g/bosc-announce) and [open-bio-l](http://mailman.open-bio.org/mailman/listinfo/open-bio-l/) mailing lists._
+
+# OBF Newsletter July/August 2022
```
Comment (peterjc) [inline newsletters/2022-07.md:9]: Maybe July/August/September now?

Or just say September and change the filename to match.

**Review by peterjc — APPROVED** [review]
Suggestion on title date, but either way best send this sooner rather than later.

**Comment by peterjc** [comment]
Added comment on #26 about sending out the emails.

File: newsletters/2022-07.md (hunk 5 above)
Comment (peterjc) [reply newsletters/2022-07.md:9]:
```suggestion
# OBF Newsletter July/August/September 2022
```

Comment (nlharris) [reply newsletters/2022-07.md:9]: makes sense

**Comment by nlharris** [comment]
I don't have merge permission (why?); can someone who does merge this?

**Comment by hlapp** [comment]
> I don't have merge permission (why?)

You weren't part of the @OBF/newsletter team. I've added you to it now.

**Pull request merged**